### PR TITLE
:fire: delete useless request and add result number on search page

### LINKED ISSUE #235: Number of results on search section
Could be nice to see how many result we have, and how many people has noted each skill

Capture d’écran de 2022-09-26 11-17-00

### PR DESCRIPTION
**Pull request by MailyLehoux**
fix #235

### CODE REVIEW

**Review by benjaminbrgs — COMMENTED** [review]
Really nice to have a count 👌🏼

I suggest to add an icon (like this one `BsFillPersonCheckFill` or another) next to the count on each skill to better understand what does the count mean.

Also, I don't know if it is wanted but when I search for a skill that which is not noted, it doesn't appear ?

**Comment by MailyLehoux** [comment]
@benjaminbrgs the icon is a good idea

Yes I don't really know. I've ntoiced that yesterday. It was an old request for this search, so I kept it. I think it will be very rare that anyone note a skill on production env, so I didn't changed that. do you think it's annoying? I don't really have an opinion on it

**Comment by benjaminbrgs** [comment]
@MailyLehoux I don't have too.. We can ignore this now, but keep it in mind if it's annoying for someone else in the future

File: src/components/SkillPanel.tsx [hunk 1]
```diff
@@ -62,8 +63,9 @@ const SkillPanel = ({
           <h2 className="text-xl">{skill.name}</h2>
           {(count || certif) && (
             <div className="flex flex-row justify-around rounded-full w-16 px-1 py-1 bg-light-med dark:bg-dark-med">
-              <div className="flex flex-col justify-center">
+              <div className="flex flex-col">
                 <span>{count}</span>
+                <BsFillPersonCheckFill />
               </div>
```
Comment (benjaminbrgs) [inline src/components/SkillPanel.tsx:69]:
```suggestion
                <span>{count}</span>
                <BsFillPersonCheckFill />
```

File: src/components/SkillPanel.tsx [hunk 2]
```diff
@@ -62,8 +63,9 @@ const SkillPanel = ({
           <h2 className="text-xl">{skill.name}</h2>
           {(count || certif) && (
             <div className="flex flex-row justify-around rounded-full w-16 px-1 py-1 bg-light-med dark:bg-dark-med">
```
Comment (benjaminbrgs) [inline src/components/SkillPanel.tsx:65]:
```suggestion
            <div className="flex flex-row items-center justify-around rounded-full w-16 px-1 py-1 bg-light-med dark:bg-dark-med">
```

**Review by benjaminbrgs — CHANGES_REQUESTED** [review]
Suggest this to horizontally align the number and the icon

image

To

image

**Comment by MailyLehoux** [comment]
thanks @benjaminbrgs looks better like this :+1:

**Review by benjaminbrgs — APPROVED** [review]

**Pull request merged**